website_product_document_download_counter/readme/CONTRIBUTORS.md

```diff
@@ -0,0 +1,5 @@
+* Cetmix OÜ \<<info@cetmix.com>\>
```

Comment:
Please use the correct way of adding the contributors. Also never add emails because they become spammed.